README.md

```diff
@@ -1 +1,165 @@
+
+This repository was created as part of my continuing education at the **Developer Academy**.
+
+## Table of Contents
+
+* [Conduit](#conduit)
+    1. [Containzation of a frontend and backend application](#contanization-of-a-frontend-and-backend-application) 
+    2. [Table of Contents](#table-of-contents)
+    3. [Prerequisites](#prerequisites)
+    4. [Description](#description)
+       1. [What is Conduit](#what-is-conduit)
+       2. [Conduit with Docker](#conduit-with-docker)
+       3. [Folder structure of the Contaization](#folder-structure-of-the-containization)
+    5. [Quickstart](#quickstart)
+    6. [Usage](#usage)
+
+
+
+## Prerequisites
+
+  * **Docker**
+  * **Compose**
+
+## Description
+
+### What is Conduit
+
+Conduit, also known as the RealWorld Example App, is an open-source example application that serves as a Medium.com-like blogging platform. 
+
+It demonstrates best practices in modern web stacks, both for the frontend (e.g. Angular, React, Vue) and the backend (e.g. Django, Node.js, Rails). 
+
+The app is often used as a reference project to learn and test complete full-stack applications with authentication, CRUD functionality and API interactions.
+
+It is provided free by [Thinkster](https://thinkster.io/)
+
+
+### Conduit with Docker
+
+Starting the frontend and backend in a container simplifies the environment configuration and makes the application portable, as all dependencies are bundled. 
+    
+It also facilitates local testing and debugging without separate servers or network configuration.
+
+
+### Folder structure of the Containization
+```
+Conduit_Deploy/
+    ├── backend
+        ├── Dockerfile
+        └── entrypoint.sh
+    ├── frontend
+        └── Dockerfile
+    ├── compose.yml
+    ├── example.env
+    └── README.md
+```
+
+## Quickstart
+
+This section gives brief and minimal instructions on how to use the setup. More in-depth instructions can be found in the [Usage](#usage) section.
+
+1. Clone the project to your platform:
+
+   ```bash
+   git clone git@github.com:cyborg-s/Conduit_Deploy.git
+   ```
+
+1b. Navigate to the project directory:
+
+   ```bash
+   cd Conduit_Deploy
+   ```
+
+2. Create the `.env` file from the template and edit your own settings:
+   
+   ```bash 
+   cp example.env .env
+   ``` 
+
+   ```bash
+   nano .env
+   ```
+
+3. Build and start the container in the background:
+
+   ```bash
+   docker compose up --build -d
+   ```
+
+
+4. Check whether the server is running correctly:
+
+    * Conduit can be accessed at the IP address of your VM on port 8282:
+
+      ```bash  
+      http://<Your_VM_IP>:8282
+      ```  
+
+    * The backend adminpanel can be accessed at the IP adress of your VM on port 2828:
+  
+      ```bash  
+      http://<Your_VM_IP>:2828/admin
+      ```  
+    
+  
+
+## Usage
+
+* Navigate to the project directory:
+
+   ```bash
+   cd Conduit_Depoyment
+   ```
+
+
+* Manage the container:
```

Comment:
should be a h3 heading in the README, the following subsequent list-elements should be h4 headings